inference-engine/src/low_precision_transformations/src/convert_subtract_constant.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "low_precision/convert_subtract_constant.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include "low_precision/network_helper.hpp"
+
+using namespace ngraph;
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::low_precision::ConvertSubtractConstant, "ConvertSubtractConstant", 0);
+
+// Original (FP16 as example, I8 in constantPrecisions):
+//
+//   Constant
+//     | I8
+//   Convert     Constant
+//      \ FP16   / FP16
+//       Subtract    Constant
+//         \ FP16   / FP16
+//          Multiply
+//
+// Result:
+//
+//   Constant    Constant
+//     | I8      | I8
+//   Convert     Convert
+//      \ FP16   / FP16
+//       Subtract    Constant
+//         \ FP16   / FP16
+//          Multiply
+//
+ngraph::pass::low_precision::ConvertSubtractConstant::ConvertSubtractConstant(const std::vector<ngraph::element::Type>& constantPrecisions) {
+    auto weightsConstantWrapper = ngraph::pattern::wrap_type<opset1::Constant>(pattern::consumers_count(1));
+    auto weightsConvertWrapper = ngraph::pattern::wrap_type<opset1::Convert>({ weightsConstantWrapper }, pattern::consumers_count(1));
+    auto subtractConstantWrapper = ngraph::pattern::wrap_type<opset1::Constant>(pattern::consumers_count(1));
+    auto subtractWrapper = ngraph::pattern::wrap_type<opset1::Subtract>({ weightsConvertWrapper, subtractConstantWrapper }, pattern::consumers_count(1));
+    auto multiplyConstantWrapper = ngraph::pattern::wrap_type<opset1::Constant>(pattern::consumers_count(1));
+    auto multiplyWrapper = ngraph::pattern::wrap_type<opset1::Multiply>({ subtractWrapper, multiplyConstantWrapper }, pattern::consumers_count(1));
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher & m) -> bool {
+        const auto& opsMap = m.get_pattern_value_map();
+        const auto weightsConvert = opsMap.at(weightsConvertWrapper).get_node_shared_ptr();
+        const auto quantizePrecision = weightsConvert->input(0).get_element_type();
+        const auto dequantizationPrecision = weightsConvert->output(0).get_element_type();
```

Comment:
Minor: perhaps it will be rather to use `get_input_element_type(0)/get_output_element_type(0)` instead of `input(0).get_element_type()/output(0).get_element_type()`